io.github.kuthulux.gnome_connection_manager.yml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.kuthulux.gnome_connection_manager
+runtime: org.gnome.Platform
+runtime-version: '44'
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --filesystem=home
+sdk: org.gnome.Sdk
+command: gnome_connection_manager.py
+modules:
+  - name: vte
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/vte/0.72/vte-0.72.1.tar.xz
+        sha256: 0554f9f88d56ce2d78398fcc7f69bc00e53bbbc5f694e0ae1dcaf5286f89d7e4
+  - shared-modules/libcanberra/libcanberra.json
+  - name: tcl
+    buildsystem: autotools
+    subdir: unix
+    post-install:
+      - chmod 755 /app/lib/libtcl8.6.so
+      - make install-private-headers
+    build-options:
+      config-opts:
+        - --enable-framework=yes
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/sourceforge/tcl/tcl-core8.6.12-src.tar.gz
+        md5: 0802e5c9e8936fc9b0c69a8da4cedfe2
+  - name: expect
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/expect/expect5.45.tar.gz
+        md5: 44e1a4f4c877e9ddc5a542dfa7ecc92b
+  - name: gcm
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/yma-het/gnome-connection-manager.git
```

Comment:
why doesn't this match the app-id?

Reply:
Because it is my fork. There is lot of fixes. When [PR](https://github.com/kuthulux/gnome-connection-manager/pull/83) will be merged, i'll switch the source.

Reply:
maybe this should wait for the upstream PR to be merged?

Reply:
it is not crucial for me, but the work is being done here is necessary anyway. if this pr must hang here before merging to original repo, so ok.